flocker/node/agents/gce.py

```diff
@@ -153,7 +170,7 @@ def _create_poller(operation):
         )
 
 
-def wait_for_operation(compute, operation, timeout_steps):
+def wait_for_operation(compute, operation, timeout_steps, sleep=None):
```

Comment:
Looks like that was lost in a refactoring, I've added it back in and its passed onto poll_until.